data/dev/repos/.gitignore

```diff
@@ -1,3 +0,0 @@
-# markus runtime data, ignore everything except .gitignore
```

Comment:
@SoftwareDev Does this file need to be removed for something on the git side to work?